source/extensions/stat_sinks/hystrix/hystrix.cc

```diff
@@ -380,7 +378,7 @@ void HystrixSink::flush(Stats::MetricSnapshot& snapshot) {
         *cluster_stats_cache_ptr, cluster_info->name(),
         cluster_info->resourceManager(Upstream::ResourcePriority::Default).pendingRequests().max(),
         cluster_info->statsScope()
-            .gaugeFromStatName(membership_total_, Stats::Gauge::ImportMode::Accumulate)
+            .gaugeFromStatName(membership_total_, Stats::Gauge::ImportMode::NeverImport)
```

Comment:
Note this was a bug caught by the integration test.

Reply:
How? Did the integration test do a hot restart?

Reply:
It hit https://github.com/envoyproxy/envoy/blob/f8cea4900b1b8be7f8f0b230a065bf4e9c311a65/source/common/stats/allocator_impl.cc#L219. I assume because of construction order? I'm not sure.

Reply:
I see...this stat is created by name here, and also established via a stat-macro in include/envoy/upstream/upstream.h:589. 

Your fix makes them declared the same way, but it seems somehow wrong that the stat is declared in two different places in the code. Are they supposed to be the same stat??

Reply:
Yes I think so. TBH I have no idea why this code is doing what it's doing. If you know of a quick fix I can make it, otherwise I would prefer to leave it to a follow up. I'm just trying to get rid of the codec hack. :)

Reply:
I think you could just put `cluster_info->stats().membership_total_ here` rather than calling the creation function. If that doesn't work we can just leave it in for now, maybe drop a todo.

Reply:
Oh and if the above cleanup works you can also drop membership_total_'s declaration and init from the HystrixSink ctor, or leave that too and we can clean up later.

Reply:
OK let me take a look and see if I can quickly fix it.

Reply:
I looked at this and I think there is a bunch of cleanup needed here since there are other stats also that should come directly from the cluster. I added a TODO.